server/src/main/java/org/elasticsearch/index/mapper/MapperRegistry.java

```diff
@@ -64,6 +64,13 @@ public MapperRegistry(Map<String, Mapper.TypeParser> mapperParsers, Map<String,
         return metadataMapperParsers7x;
     }
 
+    /**
+     * Return a map of all meta mappers that have been registered in all compatible versions.
+     */
+    public Map<String, MetadataFieldMapper.TypeParser> getAllMetadataMapperParsers() {
+        return metadataMapperParsers;
```

Comment:
I was under the impression that this held that 8.0 metadata mappers, while the other collection holds the 7.x mappers.  From the method name though, I guess I misunderstood.